### DISCUSSION TITLE
Thuật ngữ

### DISCUSSION THREAD

**Discussion by nguyenvanduocit** [vuejs-vn/docs#4] [General]
Một số thuật ngữ nên được dịch như sau:

- progressive: linh động

Một số từ nên được giữ nguyên:

- app
- object
- instance
- component
- template
- asset
- hook
- reactive

**Comment by 8bu** [comment]
1. Các từ như "expose" mình nên dịch ra là "có sẵn" hay như thế nào ạ? Có vài context sử dụng từ đó em chưa biết dịch ra tiếng Việt như thế nào.
2. `data` option trong Options API thì mình nên dịch như thế nào nhỉ. Nó là cái property `data` của Vue
```javascript
export default {
  data() {
    return {...}
  }
}
```
3. Ngoài ra có 1 số từ anh Phan An feedback em nên giữ nguyên, em xin chia sẻ với mọi người:
- code
- serve (cung cấp)
- module
- file
- server
- type

**Reply by nguyenvanduocit** [reply]
Nếu nghĩa `data` phương thức `data` của Option API thì mình nghĩ nên giữ lại.

Còn trường hợp expose thì có thể  dịch là "trình bày" không? Mà tùy context, bản dịch của mình cũn có từ này, mình dịch khác đi lắm, và mình cũng đang đợi mọi người review.

**Reply by 8bu** [reply]
À `data` thì e không dịch, ý e là cái từ **option** trong cả cụm "`data` option". Hiện tại e đang dịch là "thuộc tính `data`" không biết có đúng không.
